lib/ember_cli/shell.rb

```diff
@@ -37,6 +37,7 @@ def install
         exec "#{paths.bundler} install"
       end
 
+      exec "#{paths.ember} version || rm -rf #{paths.npm_deps} #{paths.bower_deps}"
```

Comment:
Although I wouldn't normally recommend it, since `exec` abstracts away so much, what do you think about catching the exception and returning `true`, otherwise returning `false`?